### ISSUE TITLE
Meetings with Lucinda

### ISSUE THREAD

**Issue by JonathanDavidHarris** [AguaClara/filter-constrictions#12]
## First meeting: 2/6/2018
Discussed possible ways to create the constriction of the flow cell:

- Glass beads (1mm size)
- use glue dots as grains
- hemispherical glass beads?
- vertical wires (like grass)
- varying constriction diameter to measure interception efficiency
- having multiple constrictions to look at the effects of upstream flow expansions and jets

## Meeting 2/27/2018

- We should read StaRS Theory Paper to understand more about the cylindrical annulus geometry
- Discussed issues with trying to glue sand grains onto the flow cell
- We will work on using the wire to create 2D constriction (possibly extend to make multiple in series) while still looking at other ways to create constriction

**Comment by Thomas-Bradford** [comment]
- Add pump to waste side instead of adjusting flow cell waste tube?
- Is this possible? Would it be easier than adjustable 80/20?

**Comment by Thomas-Bradford** [comment]
Calculations show that the flow rate would have to be 0.072 mL/min for one constriction. How will we obtain this velocity with our pumps?

From StaRS paper:
Total flow rate = 2ml/s
Constricted tubes = 1600
(2ml/s)/(1600 constrictions)*(60s/min) = 0.1ml/min TOO LOW

@monroews Should we consider scaling up the apparatus to be able to run the system at a more reasonable flow rate with just one constriction?

**Comment by monroews** [comment]
Your reactor design doesn't use pumps to set this flow rate!
Show the calculations. Here they are at least approximately.
here is a single constriction
I did some quick math in Python
V_filter = 1.85*u.mm/u.s
A_filter = (0.5*u.mm)**2
Q_filter = V_filter*A_filter
Q_filter.to(u.ul/u.min)= 27.75 microliter/minute
This is about 2 drops per minute!
I thought we had considered using a row of sand across your reactor.
In that case the calculations are 
V_filter = 1.85*u.mm/u.s
A_filter = 0.5*u.mm * 10*u.mm
Q_filter = V_filter*A_filter
Q_filter.to(u.ul/u.min) = 555.0 microliter/minute
Now you need to calculate the head loss in teh microbore tubing given these flow rates. If the head loss is more than a cm, then it is likely possible to control the flow by adjusting the height.

**Comment by JonathanDavidHarris** [comment]
@monroews I calculated the head loss to be 0.36 cm, with our current length of microtubing being 2.75m, so do you think it would be realistic to increase the length of our microtubing to get to 1 cm of head loss? It would probably be around 7.5m. 

With the row of sand grains, we get a head loss of about 7 cm, which is better, but the team is not sure how we would be able to control the placement of the sand grains with our current method of submerging the flow cell underwater to seal it. Currently, we are trying to get the sand grains in there without using the submerging method to see how that works, and then use the submerging method and see what happens. We will keep you posted.

**Comment by lucindali** [comment]
@monroews Is there any way to scale up the entire constriction and put a higher and more reasonable flow rate through the system? It would make the flow cell much easier to assemble, but we are concerned about sizing up the flocs and how that might change deposition.

**Comment by JacquelineDokko** [comment]
**3/20/18**

- Consider doing what StaRS does and use a pipette tip to make a constriction in the large tubing before it reaches the microtubing (needle constriction?). This might help prevent the huge flocs in the flow cell.
- Is there a way to figure out how much of the incoming flocs leave the flow cell? Turbdity?

**Comment by monroews** [comment]
I'm not sure what the goal of the pipette tip is. What is the problem you are trying to solve?

If you are trying to reduce the size of the flocs you could reduce the clay and coagulant concentrations.

**Comment by JonathanDavidHarris** [comment]
We will try to reduce the red dye and coagulant concentrations. The problem was that clogging happens right before the flocs enter the flow cell, and then large flocs form and enter the flow cell. You can see this [here](https://drive.google.com/drive/u/0/folders/1peK66c5jfxGcZKT3KkNSRkZuh8MEvzVo) at around 0:20 the monster flocs start appearing and clog the pores fast. We were trying to figure out a way to decrease the size of the flocs that enter the microtubing and Lucinda suggested we use the pipette tip like StaRS.

**Comment by monroews** [comment]
Great video. We need MUCH smaller flocs. I don't know how much flocculation you provide with tubing distance between where coagulant is injected and the flow cell. Obviously you want to decrease that distance and we need to reduce opportunities for collisions and floc growth by reducing the concentration of clay. We are trying to understand how flocs that are tiny compared with the pores are captured. 

The flocs move so quickly through the sand that is is very hard to follow them. We may need to consider higher speed photography. Is the velocity in the pores equal to the pore velocity in the StaRS filter?

**Comment by JonathanDavidHarris** [comment]
I am not sure what the pore velocity is. Since it was a row of sand, we just set the height of the head loss tubing to get the flow rate to be 555 uL per min.

**Comment by lucindali** [comment]
@monroews I suggested to break up the flocs with a construction in the tube like StaRS is doing. Even if they grow in the tubes, they can still be very small right before it goes into the flow cell.

**Comment by monroews** [comment]
Calculating the pore water velocity should be a priority. You can take the flow cell cross sectional area and multiply by 0.4 to get a good measure of the pore cross sectional area. Then use Q=VA.

**Comment by monroews** [comment]
I don't know what would be required to break up the flocs to a target size. I think that would be very difficult with the small flow rate because the size of the orifice required to generate a high shear will be smaller than the floc size. I think the better strategy is to reduce opportunities for floc growth.
